data/korath/korath missions.txt

```diff
@@ -257,3 +257,566 @@ mission "Small Scale Delivery"
+				goto check
+
+			label exile
+			action
+				set "Korath [F05]: The Rainbow Feather: exile"
+			`	"Before their exile, the Korath had a vast and thriving civilization that spanned many systems. Unfortunately, due to their very nature, they could not resolve differences easily. This led to a terrible war that left their civilization weakened.`
+			# Iris's explanation is extremely biased because her primary source
+			# is the Korath exiles. She is presenting a human interpretation of
+			# the Exiles' feelings about their own fate.
+			`	"The imperialist Drak race sent living weapons, the Archons, to conquer the Korath under the guise of protection. The Archons' antimatter weapons and seeming invincibility was too much, until the Korath gathered a larger fleet. When they managed to injure one Archon, the others revealed frightening powers. They tore open rifts in reality, simply swallowing up Korath ships. Hidden super-weapons in planetoids, intended as mines, were harmless, as the Archons shattered the planetoids and atomized the weapons. In every battle, all Korath military ships were destroyed, but the civilians were somehow unscathed.`
+			`	"The Korath saw certain defeat, and in surrender the Drak offered two options. The Korath may give up their religions and live as pets of their former friends, the Quarg. Or, they could keep their religions and live in starving exile near us.`
+			`	"The Drak left the exiles with a warning: destroy Far'en Lei, and we will kill you all. You may never leave exile until you heal that planet and live there peacefully. As a precaution, and almost as a religion, they forbid weapons on that planet. A condition YOU violated." She looks heated for a moment before continuing.`
+			`	"As to their guardianship over us, it violates the letter of the law, but not the intent. The Korath believe the Drak will look favorably upon them for protecting the weak. Along with defending us against major attacks, they have provided extensive assistance. Their aid - at our insistence - has been purely civilian: Anything we receive, the Syndicate may somehow obtain, and I shudder to think of what they may do with the wrong knowledge."`
+				goto check
+
+			label compassion
+			`	"Now that I have satisfied your curiosity, I have a request for you."`
+			choice
+				`	"Of course. What is it?"`
+				`	"I'll hear it."`
+			`	"I sense you have a part to play on the galactic stage, which I should not forbid. The rest of the galaxy knows our protectors as monsters; mindless destroyers to be hunted for thrill or profit. All I ask is that you act with compassion and understanding."`
+			choice
+				`	"You've made a convincing case. I'll help them find their place in the galaxy."`
```

Comment:
How is the player, at this point, able to make the claim that they can help an entire species find their place? This seems a bit premature.

Reply:
It is the kind of preposterous promise Iris is looking for. Hence, you have the option to give it, even if it doesn't make much sense.

Or, to put it another way: Iris has faith that [insert higher power here] will grant you the ability to manipulate the fate of the Korath, and expects you to have the same faith. This is well within the character of religious individuals I know.

Reply:
This part has been rewritten twice, so I'll resolve this conversation. If you see problems with the new text, please comment again.